src/visilibity.hpp

```diff
@@ -2051,6 +2060,71 @@ class Visibility_Graph {
 std::ostream &operator<<(std::ostream &outs,
                          const Visibility_Graph &visibility_graph);
 
```

Comment:
Doesn't have to be part of this PR, but these days I consider the superior multi-line style to be
```
std::ostream &operator<<(
  std::ostream &outs,
  const Visibility_Graph &visibility_graph,
);
```
I find that easier to read, plus it allows names on the first line to be changed without re-indentation of all subsequent lines. Can that be set in the autoformatter? If not, we don't have to worry about it.

Reply:
I will look into this. I am not really too familiar with the autoformatter, TBH. Also, I think that git/github are just showing this here for context because it's not a function that I added/rewrote in this PR. 

That said, I will definitely go back and look at the autoformatter configuration and see what can be done. Is that okay?

Reply:
If you figure it out, super. Or, if you are too busy and don't get to it, that's Ok too.

Reply:
I am going to merge this PR now but I am going to continue to fiddle with configuration for clang-format. I think that there are a few options that I can already see might be useful but getting them in the right combination is going to be tricky. I will open a separate issue for handling this, if that's okay?

Reply:
If you agree, just go ahead and resolve. Otherwise, comment away :-)